test/e2e/cli_test.go

```diff
@@ -466,7 +466,10 @@ func (s *CLISuite) TestWorkflowRetryNoPersistence() {
 		// When this is the case, this behavior is tested in cli_with_server_test.go
 		s.T().SkipNow()
 	}
+	WorkflowRetryCommon(s.E2ESuite)
+}
 
+func WorkflowRetryCommon(s fixtures.E2ESuite) {
```

Comment:
Should we be putting all these in a `cli_common.go` file?

Reply:
I don't like this. E2ESuite is an abstract base class and we shouldn't be passing it around like a normal variable.

Reply:
I think the cli_common.go is a good idea. @alexec do you have an alternate idea how to structure these common tests?